Libplanet/Address.cs

```diff
@@ -166,13 +167,45 @@ public override string ToString()
             return $"0x{ToHex()}";
         }
 
-        private static byte[] DeriveAddress(PublicKey key)
+        /// <summary>
+        /// Gets a <c>0x</c>-prefixed Mixed-case checksum address of 42 letters
+        /// that represent this <see cref="Address"/>.
+        /// </summary>
+        /// <example>A returned string looks like
+        /// <c>0x87Ae4774E20963fd6caC967CF47aDCF880C3e89B</c>.</example>
+        /// <returns>A <c>0x</c>-prefixed Mixed-case checksum address of 42
+        /// letters that represent this <see cref="Address"/>.</returns>
+        [Pure]
+        public string ToChecksumAddress()
+        {
+            var hex = ToHex();
+            var bytes = Encoding.ASCII.GetBytes(hex);
```

Comment:
Instead of re-parsing, we can call `ToByteArray()` here.

Reply:
I think the results are different. the bytes in the code is 40 bytes and the result of `ToByteArray()` is 20 bytes.

Reply:
My bad, I've mistaken.